core/src/main/java/org/bitcoinj/script/ScriptPattern.java

```diff
@@ -210,15 +210,14 @@ public static boolean isSentToMultisig(Script script) {
         try {
             // Second to last chunk must be an OP_N opcode and there should be that many data chunks (keys).
             ScriptChunk m = chunks.get(chunks.size() - 2);
-            if (!m.isOpCode()) return false;
```

Comment:
Hmmm, rather than continuing to rely on an exception flow why not add an `isOpN()` and use that? And then remove the try/catch.

Reply:
Good idea. Will do this.